estuary/utils/story.py

```diff
@@ -276,6 +278,42 @@ def _get_plural_label(label):
             raise RuntimeError('{0} does not have a {1} relationship'.format(
                 story_node_display_name, rel_direction))
 
+    def get_recent_nodes(self):
+        """
+        Get the most recent nodes' IDs.
+
+        :param str label: Neo4j node label
+        :param int num_recents: the number of most recent nodes to return
+        :return: array of the most recent nodes
+        :rtype: array
+        """
+        label_dict = {
+            FreshmakerEvent.__label__: 'id',
+            BugzillaBug.__label__: 'modified_time',
+            DistGitCommit.__label__: 'commit_date',
+            KojiBuild.__label__: 'completion_time',
+            Advisory.__label__: 'update_date'
+        }
+
+        query = 'CALL apoc.cypher.runMany(\''
+        for label, time_property in label_dict.items():
+            query += (
+                '\n    MATCH ({label}:{label}) RETURN {label} '
+                'ORDER BY {label}.{time_property} DESC LIMIT 5;'
+            ).format(label=label, time_property=time_property, rv=label.lower())
+        query = '{0}\'\n, {{}}, {{statistics: False, timeout: 15}});'.format(query)
```

Comment:
Also, what happens when the timeout is reached when running this query from neomodel? Do we need error handling for that?

Reply:
Everything else should be good, but I am unsure about this still